.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Reason for change
+
+<!-- Explain what this PR does -->
+
+---
+
+## Reminders
+
+- Update package version number
+- Frontend accessibility check
+- Styleguide example in place
+- New functionality covered by tests
+- Screenshots of multiple screen sizes
+
+---
+
+## Commit Messages
+
+<!-- Paste your commit messages below. Run the following in your terminal to copy them: -->
```

Comment:
Do we need this? The commits should all be referenced in the PR screen itself to take care of the evolution of the code.